config/locales/fr.yml

```diff
@@ -177,29 +177,29 @@ fr:
         text: Besoin d'un nouveau mot de passe ? Pas de problème. %{link}
         title: Renvoi de mot de passe
       delete:
-        delete:
-        delete_profile:
-        warning:
+        delete: Supprimer
+        delete_profile: Supprimer le profil
+        warning: La suppression de votre profil est une action irréversible. Cela ne peut pas être annulé en contactant le support. Soyez sûr !
       mfa:
-        multifactor_auth:
-        disabled:
-        go_settings:
-        enabled:
-        disable:
+        multifactor_auth: Authentication Multifacteur
+        disabled: Vous n'avez pas encore activer l'authentication multifacteur.
+        go_settings: Enregistrer un nouvel appareil
+        enabled: Vous avez activé l'authentication multifacteur. Veuillez entrer l'OTP de votre authentificateur ou un de vos codes de récupération pour le désactiver.
+        disable: Désactiver
     delete:
-      title:
-      confirm:
-      instructions:
-      list_only_owner:
-      list_multi_owner:
+      title: Supprimer le profil
+      confirm: Confirmer
+      instructions: Nous sommes désolés de vous voir partir. Veuillez entrer votre mot de passe dans le champ et confirmer.
+      list_only_owner: Ces gems seront effacées quand vous effacerez votre profil. Si vous désirez ajouter des propriétaires avant de supprimer votre profil, vous pouvez utiliser la commande %{command_link}.
```

Comment:
Woah do I understand that gems with a single owner will be deleted if the last owner deletes his/her account? I need to dive into this a bit further :smile:

Reply:
It seems like so 🤨
![image](https://user-images.githubusercontent.com/22852543/47227292-c8284a80-d3c2-11e8-9b0c-5c0d30487028.png)

http://help.rubygems.org/discussions/questions/8961-delete-my-account

Reply:
See also: https://github.com/rubygems/rubygems.org/pull/1301#discussion_r70223143